models/aave/ethereum/aave_v2_supply.sql

```diff
@@ -0,0 +1,124 @@
+{{ config(
+  materialized='view'
+  , alias='supply'
+  , post_hook='{{ expose_spells(\'["ethereum"]\',
+                                  "project",
+                                  "aave_v2",
+                                  \'["batwayne", "chuxinh"]\') }}'
+  )
+}}
+
+SELECT 
+      version,
+      transaction_type,
+      erc20.symbol,
+      deposit.token as token_address, 
+      depositor,
+      withdrawn_to,
+      liquidator,
+      amount / concat('1e',erc20.decimals) AS amount,
+      (amount / concat('1e',p.decimals)) * price AS usd_amount,
+      evt_tx_hash,
+      evt_index,
+      evt_block_time,
+      evt_block_number 
+FROM (
+ SELECT 
+    '1' AS version,
+    'deposit' AS transaction_type,
+    CASE
+        WHEN _reserve = '\xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE' THEN '\xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' 
+        ELSE _reserve
+    END AS token,
+    _user AS depositor, 
+    NULL::string AS withdrawn_to,
+    NULL::string AS liquidator,
+    _amount AS amount,
+    evt_tx_hash,
+    evt_index,
+    evt_block_time,
+    evt_block_number
+FROM {{ source('aave_ethereum','LendingPool_evt_Deposit') }}
+UNION ALL 
+SELECT 
+    '1' AS version,
+    'withdraw' AS transaction_type,
+    CASE
+        WHEN _reserve = '\xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE' THEN '\xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' 
```

Comment:
Addresses on v2 have to follow this format : `'0x...'` instead of `'\x...'`, this is valide for all addresses in all models

Reply:
Addresses also have to be lowercase

Reply:
Done